src/commands/asset-compute/index.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const HHelp = require('@oclif/plugin-help').default;
```

Comment:
random question: why `HHelp` instead of `Help`? Does the extra H stand for something?

Reply:
IDK. I just copy pasted this from [aio-cli-plugin-app](https://github.com/adobe/aio-cli-plugin-app/blob/master/src/commands/app/index.js).